backend/models/mongo_models.py

```
@@ -957,3 +957,132 @@ def to_dict(self):
+        if not version:
+            # Create initial version
+            import uuid
+
+            initial_count = 0
+
+            # Get initial count if model class is provided
+            if model_class:
+                try:
+                    initial_count = model_class.objects().count()
+                except Exception:
+                    pass  # Use default 0 if count fails
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)